src/main/webapp/renderers/Step3Renderer.js

```diff
@@ -9,6 +9,13 @@ export default class Step3Renderer extends RendererInterface {
         return '#'; // TODO(tzavidas): create the actual link
     }
 
+    formatDate(date) {
```

Comment:
This comment is not applicable anymore, as all renderers will accept time in the string format of "HH:MM". The Date object will only be used when performing API calls. This object will have the same time, but it will be 7 days into the future!